auth/sql/change-username-collation.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE users MODIFY COLUMN username varchar(255) NOT NULL COLLATE utf8mb4_0900_as_cs;
+ALTER TABLE users MODIFY COLUMN login_id varchar(255) NOT NULL COLLATE utf8mb4_0900_as_cs;
```

Comment:
you promoted this from `null` to `not null`, I think it will pass if you switch back to `NULL`.

Reply:
it needs a `DEFAULT NULL` actually